### PR TITLE
react-credit-cards: Export "Focused" type to make values available externally

### PR DESCRIPTION
**Pull request by olefrank**

### CODE REVIEW

**Review by vstrimaitis — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

File: types/react-credit-cards/index.d.ts [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
+// Definitions by: Vytautas Strimaitis <https://github.com/vstrimaitis>, Ole Frank <https://github.com/olefrank>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import * as React from "react";
 
 export interface CallbackArgument {
     isValid: boolean;
-    type: { issuer: string; maxLength: number; };
+    type: { issuer: string; maxLength: number };
```
Comment (olefrank) [inline types/react-credit-cards/index.d.ts]: Why remove semi colon?

File: types/react-credit-cards/index.d.ts [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
 export interface ReactCreditCardProps {
     acceptedCards?: ReadonlyArray<string>;
     callback?: (type: CallbackArgument, isValid: boolean) => void;
     cvc: string | number;
     expiry: string | number;
-    focused?: "name" | "number" | "expiry" | "cvc";
+    focused?: Focused;
     issuer?: string;
-    locale?: { valid: string; };
+    locale?: { valid: string };
```
Comment (olefrank) [inline types/react-credit-cards/index.d.ts]: Why remove semi colon?

File: types/react-credit-cards/index.d.ts [hunk 3]
```diff
@@ -1,31 +1,35 @@
...
     name: string;
     number: string | number;
-    placeholders?: { name: string; };
+    placeholders?: { name: string };
```
Comment (olefrank) [inline types/react-credit-cards/index.d.ts]: Why remove semi colon?

**Pull request merged**